src/component/tooltip/TooltipHTMLContent.ts

```diff
@@ -355,6 +354,7 @@ class TooltipHTMLContent {
         el.style.pointerEvents = this._enterable ? 'auto' : 'none';
 
         this._show = true;
+        this._firstShow && (this._firstShow = false);
```

Comment:
I think this condition is not necessary. The expression can be simply
```js
this._show = true;
this._firstShow = false;
```

Reply:
Force of habbit :smile